src/lib/y2storage/partitionable.rb

```diff
@@ -43,6 +43,8 @@ class Partitionable < BlkDevice
     #   @return [Array<PartitionTables::Type>]
     storage_forward :possible_partition_table_types, as: "PartitionTables::Type"
 
+    storage_forward :default_partition_table_type, as: "PartitionTables::Type"
+
```

Comment:
This method and `possible_partition_table_types` need documentation. Specially when compared to `preferred_ptable_type` (that is, btw, inserted in a different point of the hierarchy)